docs/configure_rum_android_sdk.md

```diff
@@ -80,9 +80,56 @@ In addition to [tracking actions automatically][5], you can also track specific
 {{% /tab %}}
 {{< /tabs >}}
 
+### Enrich resources
+
+When [tracking resources automatically][6], provide a custom `RumResourceAttributesProvider` instance to add custom attributes to each tracked network request. For example, if you want to track a network request's headers, create an implementation as follows, and pass it in the constructor of the `DatadogInterceptor`.
+
+{{< tabs >}}
+{{% tab "Kotlin" %}}
+```kotlin
+class CustomRumResourceAttributesProvider : RumResourceAttributesProvider {
+    override fun onProvideAttributes(
+        request: Request,
+        response: Response?,
+        throwable: Throwable?
+    ): Map<String, Any?> {
+        val headers = request.headers()
+        return headers.names().associate {
+            "headers.${it.lowercase(Locale.US)}" to headers.values(it).first()
+        }
+    }
+}
+```
+{{% /tab %}}
+{{% tab "Java" %}}
+```java
+
+public class CustomRumResourceAttributesProvider implements RumResourceAttributesProvider {
+    @NonNull
+    @Override
+    public Map<String, Object> onProvideAttributes(
+            @NonNull Request request,
+            @Nullable Response response,
+            @Nullable Throwable throwable
+    ) {
+        Map<String, Object> result = new HashMap<>();
+        Headers headers = request.headers();
+
+        for (String key : headers.names()) {
+            String attrName = "headers." + key.toLowerCase(Locale.US);
+            result.put(attrName, headers.values(key));
```

Comment:
java snippet doesn't exactly match kotlin snippet: in Java we add all the values for the header name, in kotlin we add only a single value.

Reply:
Fixed